frontend/src/components/publication/PublicationActions.vue

```diff
@@ -60,16 +81,83 @@ const withdrawDocUnit = async () => {
   docUnitPublicationError.value = error ?? null
   await store.loadDocumentUnit(decision.value.documentNumber)
   isWithdrawing.value = false
+  isPublished.value = false
+  isWithdrawn.value = error == null
 }
+
+const lastPublishedAt = computed(() => {
+  if (decision.value.managementData.lastPublishedAtDateTime) {
+    return DateUtil.formatDateTime(
+      decision.value.managementData.lastPublishedAtDateTime,
+    )
+  }
+  return "-"
+})
 </script>
 
 <template>
   <div class="flex flex-col gap-24">
     <div class="flex flex-col gap-16">
+      <InfoModal
+        v-if="isUat"
+        aria-label="Information Testportal in UAT"
+        class="mt-8"
+        v-bind="uatMessage"
+        :link="uatLink"
+        :status="InfoStatus.INFO"
+      />
       <h3 class="ris-label1-bold">Aktueller Status Portal</h3>
       <PortalPublicationStatusBadge
         :status="decision.portalPublicationStatus"
       />
+      <div
+        v-if="
+          decision.portalPublicationStatus ===
+            PortalPublicationStatus.PUBLISHED ||
+          decision.portalPublicationStatus === PortalPublicationStatus.WITHDRAWN
+        "
+        class="flex flex-row items-center gap-8"
+      >
+        <IconErrorOutline class="text-grey-900" />
+        <a
+          class="ris-link1-regular whitespace-nowrap no-underline focus:outline-none focus-visible:outline-4 focus-visible:outline-offset-4 focus-visible:outline-blue-800"
+          :href="linkToPortal"
+          rel="noopener noreferrer"
+          target="_blank"
+        >
+          Portalseite der Dokumentationseinheit
+        </a>
+        <div
+          v-if="
+            decision.portalPublicationStatus ===
+            PortalPublicationStatus.WITHDRAWN
+          "
+        >
+          wurde entfernt
+        </div>
+      </div>
+
+      <div v-if="isPublished || isWithdrawn" class="flex flex-row gap-8">
+        <IconErrorOutline class="text-grey-900" />
+        <p>
+          Das Hochladen der Stammdaten und der Informationen im Portal-Tab
+          „Details“ kann bis zu 2 Minuten dauern.
```

Comment:
```suggestion
          „Details“ dauert etwa 2 Minuten.
```
Ich denke, das sollten wir vorsichtiger formulieren. Wenn gerade viel verarbeitet wird, dauert es länger.